content/kb/troubleshooting-rwx-mount-ownership-set-to-nobody.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Troubleshooting: Longhorn RWX shared mount ownership is shown as nobody in consumer Pod"
+author: Chin-Ya Huang
+draft: false
+date: 2021-03-31
+categories:
+  - "rwx"
+---
+
+## Applicable versions
+
+All Longhorn versions.
+
+## Symptoms
+
+When Pod mounts with RWX volume, the Pod share mount directory and all of the ownership of its recurring contents are shown as nobody, and in the share-manager is shown as root.
+
+```
+root@ip-172-30-0-139:/home/ubuntu# kubectl exec -it rwx-test-2pml2 -- ls -l /data
+total 16
+drwx------ 2 nobody 42949672 16384 Mar 31 04:16 lost+found
+
+root@ip-172-30-0-139:~# kubectl -n longhorn-system exec -it share-manager-pvc-f3775852-1e27-423f-96ab-95ccd04e4777 -- ls -l /export/pvc-f3775852-1e27-423f-96ab-95ccd04e4777
+total 16
+drwx------ 2 root root 16384 Mar 31 04:42 lost+found
+```
+
+## Background
+
+The nfs-ganesha in share-manager uses idmapd for NFSv4 ID mapping and is set to use `localdomain` as its export Domain.
+
+## Reason
+
+A result of content mismatch in /etc/idmapd.conf between client(share-manager) and server(host) cause ownership to change.
```

Comment:
Client is host, server is share manager pod.

Reply:
Thanks, fixed.